tests/installation/boot_into_snapshot.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
I see you are adding completely new file. How does these © holds up then?
Just `# Copyright © 2016 SUSE LLC` I would say.

@okurz IIRC you did some updates to licenses, are they everywhere like this now? Should we use this for new tests as if they relates to whole test suite?